readme.md

```diff
@@ -60,16 +60,25 @@ In short, the notion of which variables are available where.
 ---
 
 <details>
-<summary>So far in class, what is the primary way to control scope in JS?</summary>
+<summary>So far in class, what is the primary way to control scope in JS? Why do we want to control scope?</summary>
 
-Through the use of functions to create new local scopes.
+Functions create new local scopes.
+
+Scope allows us to use generic variable names (eg `count`, `input`, `evt`) while considering them in an isolated space.
+This makes our code easier to reason about.
 
 </details>
 
 #### `let`
 
-So far, the primary way to control scope in an application has been through the use
-of functions:
+The fact that other blocks (`if`, `for`, `while`, etc) *do not* create scopes is one of the oddities for which JS does and should get knocked.
+
+ES6 introduces the `let` keyword which works just like `var` but is scoped to its block (`{...}`) rather than its function.
+
+> Objects use curly braces `{}` but are not code blocks!
```

Comment:
Maybe this? `All the object literals that we've created so far use curly braces`...